docs/website/docs/development/architecture/how-odo-works.md

```diff
@@ -82,6 +82,7 @@ By default, `odo` adds the following annotations to the Deployment:
 | Key                                    | Description                                                                                                                                                                                                 | Example Value       |
 |----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
 | `odo.dev/project-type `                | the application runtime, if available. Value is read in order from the `metadata.projectType` or `metadata.language` fields in the Devfile. As both metadata are optional, this annotation can be omitted.  | `spring`            |
+| `alpha.image.policy.openshift.io/resolve-names` | Enable the use of ImageStreams on OpenShift | `*` |
```

Comment:
Can you also add this to the concrete example provided just below this table?

![image](https://user-images.githubusercontent.com/593208/207570119-1e292507-4f75-4814-9300-061861c7c696.png)